core/services/vrf/proof/seed.go

```diff
@@ -45,6 +45,17 @@ type PreSeedDataV2 struct {
 	Sender           common.Address
 }
 
+type PreSeedDataV2Plus struct {
```

Comment:
I'm not sure why the `PreSeed*` objects are even needed, because the final seed is always just the preseed + the blockhash, and not the rest of the stuff.